src/lib/moxiedoc.ts

```diff
@@ -95,6 +97,8 @@ const process = (settings: MoxiedocSettings): MoxiedocResult => {
     }
   });
 
+  builder.api.setStructure(settings.structure);
```

Comment:
As noted previously, this shouldn't be done like this... we should be passing this to the `Exporter` below instead and then passing it to the template render functions.